src/hugr/rewrite/outline_cfg.rs

```diff
@@ -163,31 +171,37 @@ impl Rewrite for OutlineCfg {
             h.move_before_sibling(new_block, outer_entry).unwrap();
         }
 
-        // 5. Children of new CFG.
-        // Entry node must be first
-        h.move_before_sibling(entry, inner_exit).unwrap();
-        // And remaining nodes
-        for n in self.blocks {
-            // Do not move the entry node, as we have already
-            if n != entry {
-                h.set_parent(n, cfg_node).unwrap();
+        {
+            // These operations do not fit into any SiblingView
```

Comment:
I guess I mean SiblingMut. So we'd need a full-on DescendantsMut for that. I think I'm happy to skip for now but I could probably be persuaded....